### PR TITLE
[Doc] Add video tutorial for List and Edit pages

### PR DESCRIPTION
**Pull request by fzaninotto**
React-admin now has [video tutorials on YouTube](https://www.youtube.com/@react-admin)! To make them more discoverable, we embed them in the related page. This should help newcomers who don't like to read docs.

### CODE REVIEW

File: docs/Edit.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: "The Edit Component"
 
 The `<Edit>` component is the main component for edition pages. It fetches a record based on the URL, prepares a form submit handler, and renders the page title and actions. It is not responsible for rendering the actual form - that's the job of its child component (usually a form component, like [`<SimpleForm>`](./SimpleForm.md)). This form component uses its children ([`<Input>`](./Inputs.md) components) to render each form input.
 
-![post edition form](./img/edit-view.png)
```
Comment (slax57) [inline docs/Edit.md:10]: why not keep the picture in the 'Usage' section, just like you did for List?

**Comment by slax57** [comment]
It renders poorly on mobile...


https://github.com/marmelab/react-admin/assets/14542336/ca7a3a0e-1284-46dc-825f-d675fadd5c12

**Comment by fzaninotto** [comment]
> It renders poorly on mobile...
> 
>  Capture.video.2024-01-17.18.27.23.mp4

I know, but that's the default Youtube embed code. I haven't found a better way.

File: docs/Edit.md (hunk 1 above)
Comment (fzaninotto) [reply docs/Edit.md:10]: Because this (outdated) screenshot doesn't match the code in the usage section.

**Review by slax57 — APPROVED** [review]

**Pull request merged**